### ISSUE TITLE
investigate: VPN counters on InfluxDB USG dashboard

### ISSUE THREAD

**Issue by lslamp** [unpoller/unpoller#282]
David,
Firstly, let me say thank you very much for what you have done. IT is an exceptional project. I am very impressed with what you have done. I have searched for many months trying to find a better monitoring system that what the unifi product has delivered.
I hope that these questions are suitable and acceptable here. IF not please just delete them.
1. How are you affiliated with unifi, do you work for them?
2. Looking at the template Unifi-Poller: USG Insights - InfluxDB:- looking at the below image, on the bottom left there is a field above localdomain, where is that sourced from? How can I populate that with icanhome.e2snail.com?
![image](https://user-images.githubusercontent.com/6013759/100551867-c6106580-3283-11eb-8c95-2357651137d2.png)
3. Looking at the template Unifi-Poller: Network Sites - InfluxDB:- looking at the screenshot below, you can see VPN Users. I have a VPN configured, but when I connect, I do not see that I am connected, what I do see if I look at the history 3 days ago, theer are 2 users that show as disconnected, only I cannot who these users are.
How can I see what VPN users are connected, and why when I connect, nothing is shown in the panel?
![image](https://user-images.githubusercontent.com/6013759/100552047-fad0ec80-3284-11eb-871f-9a5d74b8d4c8.png)

Then lastly, how are you compensated for this great work that you have done. is there any reason why your work is not visible on the unifi community site. Can I make them aware of your great work?

Thanks
Lawrence

**Comment by davidnewhall** [comment]
Hello! Thank you for the kind words. :)

1. I'm not affiliated with UniFi. They pay no attention to me.
2. Sounds like you're new to Grafana. Welcome!
3. Off hand, I have no idea why your VPN count isn't changing. I can ask many questions to figure it out, but that may be easier in [Discord](https://discord.gg/KnyKYt2). I don't have VPN users, so it's always 0 for me, and it's possible it's not displaying the correct value. As far as displaying who's connected.. that's possible, but since I don't have VPN users I have no interest in maintaining a VPN-user panel. You can probably make one with the InfluxDB datasource in Grafana though. 

tl;dr: Everything is customizable. You're going to have to start clicking the _edit panel_ buttons in Grafana and figuring out what all that stuff is. I'm sorry but I don't have time to teach those skills; plenty of info in the Internet about how to use Grafana.

4. I'm not compensated for this project. I wrote it for myself, and other folks really like it too.
5. I'm not sure what you're referring to by "unifi community site" - sounds like the forums. If you know of a way to make this app more visible, go for it!

**Comment by lslamp** [comment]
I have created a new topic on the community, but also I found that someone else has also been promoting your templates.
As for being new to Grafana, good guess, I have heard about it but installing your templates is my very first experience with it.

As for the VPN issue, I will take a look on Discord and see what I can find.
Hope you get more visibility.

Thanks again for a great product.
Lawrence

**Comment by davidnewhall** [comment]
>  I found that someone else has also been promoting your templates.

Got a few of those nice folks. :)

> As for the VPN issue, I will take a look on Discord and see what I can find.

I meant connect to the discord, join the #unifi-poller channel and lets talk. For now I'll leave this open to remind me check the vpn user counters to maker sure I don't do anywhere funky with 'em.

**Comment by lslamp** [comment]
David, was the channel on Discord. I am not sure if I am doing something wrong, but I cannot find teh channel?

**Comment by davidnewhall** [comment]
Sorry missed this. This is the link to the Ubiquiti Discord: https://discord.gg/KnyKYt2
You'll find me in the #unifi-poller channel.

**Comment by davidnewhall** [comment]
Okay, sorry for the delay. I finally had time to look at this. If you edit the VPN counter panel, it's currently set to `field(num_user)`. Change that to `field (remote_user_num_active)` as shown here:
![Screen Shot 2021-03-22 at 2 07 02 AM](https://user-images.githubusercontent.com/2402929/111965823-6b5d7c80-8ab3-11eb-899b-df682cb64991.png)

If that doesn't work, try some of the other values that show up in that menu and see if any of them show the proper number of users connect via VPN. I have a feeling the change above will fix this though. Please let me know so I can get this into the next dashboard update.

Thanks!

**Comment by lslamp** [comment]
@platinummonkey @davidnewhall 
Where can I get help / advice on the Grafana Angular issue that is showing on the unifi-poller.

Right now.

Lawrence
